#input topic info, (yaw,pitch,roll in degree/ x,y,z in meter)
CloudIn1:
  topic_name: /pcl_front
  pose_yaw: 0
  pose_pitch: 0
  pose_roll: 0
  pose_x: 0
  pose_y: 0
  pose_z: 0
CloudIn2:
  topic_name: /pcl_left
  pose_yaw: 0
  pose_pitch: 0
  pose_roll: 0
  pose_x: 0
  pose_y: 0
  pose_z: 0
CloudIn3:
  topic_name: /pcl_right
  pose_yaw: 0
  pose_pitch: 0
  pose_roll: 0
  pose_x: 0
  pose_y: 0
  pose_z: 0
CloudIn4:
  topic_name: /pcl_rear
  pose_yaw: 0
  pose_pitch: 0
  pose_roll: 0
  pose_x: 0
  pose_y: 0
  pose_z: 0
#output topic info (frame_id for virtual frame)
CloudOut:
  topic_name: pcl_merge
  frame_id: base
